rocAL/rocAL/include/readers/image/reader.h

```diff
@@ -46,6 +46,7 @@ enum class StorageType
     SEQUENCE_FILE_SYSTEM = 6,
     MXNET_RECORDIO = 7,
     VIDEO_FILE_SYSTEM = 8,
+    TEXT_FILE_SYSTEM = 9,
 };
```

Comment:
What is this TEXT_FILE_SYSTEM?
is this change needed in this PR?

Reply:
For reading txt files - i.e the txt files that conatins the audio file names seperated by labels